server/schedulers/hot_test.go

```diff
@@ -845,7 +845,7 @@ func (s *testHotReadRegionSchedulerSuite) TestWithPendingInfluence(c *C) {
 			// store byte/key rate (min, max): (6.6, 7.1) | 6.1 | (6, 6.5) | 5
 
 			op2 := hb.Schedule(tc)[0]
-			testutil.CheckTransferPeerWithLeaderTransfer(c, op2, operator.OpHotRegion, 1, 4)
+			testutil.CheckTransferPeer(c, op2, operator.OpHotRegion, 1, 4)
```

Comment:
Why

Reply:
As we have supported hot read peer scheduling, the hot read scheduling may not always transfer leader peer.